server/src/main/java/au/org/aodn/ogcapi/server/core/service/OGCApiService.java

```diff
@@ -78,13 +78,16 @@ public String processDatetimeParameter(String datetime, String filter) {
 
         // for now, assumption is that temporal is the only filter
         if (filter == null) {
-            if (datetime.contains("/")) {
+            if (datetime.contains("../")) {
```

Comment:
should be datetime.startsWith("../") || datetime.startsWith("/"), 
same idea applies to the reset

Reply:
for endsWith ('/') || endsWith ("/..")